ocs_ci/ocs/quay_operator.py

```diff
@@ -47,13 +47,41 @@ def __init__(self):
             else constants.DEFAULT_STORAGECLASS_RBD
         )
 
-    def setup_quay_operator(self):
+    def get_quay_default_channel(self):
```

Comment:
you can get default channel from here: https://github.com/red-hat-storage/ocs-ci/blob/f87828ce9fd737524f5b49b42fd63affccc6f012/ocs_ci/ocs/resources/packagemanifest.py#L101

Reply:
Done